commands/i18n.js

```diff
@@ -1,42 +1,41 @@
-import configuration from "../lib/config.js";
-import { Local } from "../lib/resolvers/local.js";
-import { Core } from "../lib/resolvers/core.js";
-import { Extensions } from "../lib/resolvers/extensions.js";
-import { State } from "../lib/state.js";
+import configuration from '../lib/config.js';
+import { Local } from '../lib/resolvers/local.js';
+import { Core } from '../lib/resolvers/core.js';
+import { Extensions } from '../lib/resolvers/extensions.js';
+import { State } from '../lib/state.js';
+import { i18n } from '../api/i18n.js';
 
-import { i18n } from "../api/i18n.js";
+export const command = 'i18n <command>';
 
-export const command = "i18n <command>";
+export const aliases = ['i'];
 
-export const aliases = ["i"];
+const [extract, compile] = ['extract', 'compile'];
 
-const [extract, compile] = ["extract", "compile"];
-
-export const describe = "Extract and compile i18n strings for the podlet.";
+export const describe = 'Extract and compile i18n strings for the podlet.';
 
 export const builder = (yargs) => {
-  yargs.example("podlet i18n [command]");
+  yargs.example('podlet i18n [command]');
 
-  yargs.positional("command", {
-    type: "string",
+  yargs.positional('command', {
+    type: 'string',
     default: extract,
     choices: [extract, compile],
-    describe: "i18n command to run",
+    describe: 'i18n command to run',
   });
 
   return yargs.argv;
 };
 
 export const handler = async (argv) => {
-  const { command } = argv;
+  const { command: commandArgument } = argv;
```

Comment:
Why is it changing this?

Reply:
It collided with the `command` declared in line 8 in the same file.